dockers/docker-dhcp-relay/cli-plugin-tests/test_config_dhcp_relay.py

```diff
@@ -0,0 +1,175 @@
+import os
+import sys
+import traceback
+from unittest import mock
+
+from click.testing import CliRunner
+
+from utilities_common.db import Db
+
+import pytest
+
+sys.path.append('../cli/config/plugins/')
+import dhcp_relay
+
+@pytest.fixture()
+def mock_cfgdb():
+    cfgdb = mock.Mock()
+    CONFIG = {
+        'VLAN': {
+            'Vlan1000': {
+                'dhcp_servers': ['192.0.0.1']
+            }
+        }
+    }
+
+    def get_entry(table, key):
+        return CONFIG[table][key]
+
+    def set_entry(table, key, data):
+        CONFIG[table].setdefault(key, {})
+        CONFIG[table][key] = data
+
+    cfgdb.get_entry = mock.Mock(side_effect=get_entry)
+    cfgdb.set_entry = mock.Mock(side_effect=set_entry)
+
+    yield cfgdb
+
+
+config_vlan_add_dhcp_relay_output="""\
+Added DHCP relay destination address 192.0.0.100 to Vlan1000
+Restarting DHCP relay service...
+"""
+
+config_vlan_del_dhcp_relay_output="""\
```

Comment:
I think `\` is not required here.

Reply:
\ here is to not have newline as first character in this string otherwise this expected output will not match exactly.